eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -113,7 +113,7 @@ jobs:
         displayName: "Publish Report Artifacts"
         inputs:
           artifactName: reports
-          path: $(Build.ArtifactStagingDirectory)/dependencies.html
+          path: $(Build.ArtifactStagingDirectory)/*
```

Comment:
What else is in this directory?

Reply:
it should only contain two files dependencies.html, and data.js, that's assuming the directory starts out as empty for every job. I'll find out in a test run that I have queued.